src/inject/inject.js

```diff
@@ -14,16 +14,33 @@ chrome.extension.sendMessage({}, function(response) {
 			bar.style.backgroundColor = "dodgerblue";
 			bar.style.textAlign = "center";
 			bar.style.position="fixed"; 
-
 			bar.style.padding = "2px";
 			bar.style.zIndex = "10000";
 			bar.style.borderRadius = "5px";
 
+		var overlay = document.createElement("div");
+			overlay.setAttribute("id", "overlay");
+			overlay.style.width = "100%";
+			overlay.style.height = "100%";
+			overlay.style.background = "rgba(255,255,255,0.8)";
+			overlay.style.top = "0";
+			overlay.style.left = "0";
+			overlay.style.position = "absolute";
+			overlay.style.zIndex = "9999";
+
+		var source = document.createElement("source");
+			source.src = chrome.extension.getURL("src/assets/sounds/alert.ogg");
+			source.type = "audio/ogg";
+			
+		var audio = document.createElement("audio")
+			audio.setAttribute("loop", "true");
```

Comment:
I also think it is invasive in the audio loop and the overlay. Both should be optional.